setup.py

```diff
@@ -54,6 +55,7 @@ def read(fname):
         # gives the C++ SymEngine library, while conda-forge/python-symengine
         # provides the Python package called `symengine`.
         'importlib_metadata',  # drop when pycalphad drops support for Python<3.8
+        'importlib_resources',  # drop when pycalphad drops support for Python<3.9
```

Comment:
Should also be added to `requirements-dev.txt`

Reply:
Currently `requirements-dev.txt` only has the _additional_ dependencies required for building the package and building the documentation, the runtime requirements are not included and everything in the setup.py install requires is installed if users follow our development version install procedure. I don't think this is a build time requirement (otherwise our CI should be telling us that by needing this package in our pyproject.toml build system requirements).

Reply:
Oh, okay.